h/static/scripts/group-forms/components/test/CreateEditGroupForm-test.js

```diff
@@ -128,90 +103,6 @@ describe('CreateEditGroupForm', () => {
     wrappers.forEach(wrapper => wrapper.unmount());
   });
 
-  [
```

Comment:
Testing of the `minLength` and `maxLength` props is now done in the `TextField` tests. CreateEditGroupForm now just sets static values for these.